lib/frames.rb

```diff
@@ -0,0 +1,77 @@
+require './lib/score_card'
+require './lib/roll'
+
+class Frames
```

Comment:
You normally want to keep class names singular, as plural indicates a list. Frame doesn't quite describe what this class is doing, can you think of an alternative singular name?